lib/hanami.rb

```diff
@@ -110,7 +125,7 @@ def self.app_path(dir = Dir.pwd)
     path = dir.join(APP_PATH)
 
     if path.file?
-      path.to_s
+      path
```

Comment:
This changes public API. Are we sure we want this to be a pathname? (I'm always in favor of pathnames FWIW).

Reply:
Yep, I did consider this, and IMO this is a low risk API change and has the benefit of consistency with the other places we return pathnames, like config.root.

Reply:
FWIW I doubt anyone is calling .app_path directly yet anyway. We added it relatively recently to make it so you could call the Hanami CLI in deeply nested directories and have everything still work.

Reply:
Yes I agree